Cargo.toml

```diff
@@ -1,16 +1,26 @@
 [package]
-name = "fitbit-grabber"
-version = "0.1.0"
 authors = ["Bradford D. Boyle <bradford.d.boyle@gmail.com>"]
+autobins = false
 license = "Apache-2.0"
+name = "fitbit-grabber"
+version = "0.1.0"
 
 [dependencies]
-clap = "2.31"
 chrono = "0.4"
+clap = "2.31"
+failure = "0.1.1"
 oauth2 = "1.3"
 reqwest = "0.8"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 tiny_http = "0.6"
+toml = "*"
```

Comment:
It should have a version, for sure.

I actually backed out a lot more cli stuff, but left this in since it didn't seem disruptive, and perhaps useful. I made a new repo for a more daemony thing, and specified _this_ repo as a local path dependency: https://github.com/anxiousmodernman/fitbit-syncd/blob/master/Cargo.toml#L26

In the long run, separate crates for bins and libs is probably the way to go. But we can keep things co-mingled here for a while.